lighthouse-cli/run.js

```diff
@@ -8,6 +8,7 @@
 /* eslint-disable no-console */
 
 const path = require('path');
+const psList = require('ps-list');
```

Comment:
doing this in a crossplatform way is a gigantic pain, a CLI-only dep that's pretty small seems ok for this